frontend/packages/data-portal/app/components/common/CustomBanner.tsx

```diff
@@ -0,0 +1,62 @@
+import { Banner, Icon } from '@czi-sds/components'
+import { useLocalStorageValue } from '@react-hookz/web'
+import dayjs, { OpUnitType } from 'dayjs'
+import { useState } from 'react'
+
+import { LocalStorageKeys } from 'app/constants/localStorage'
+import { useEffectOnce } from 'app/hooks/useEffectOnce'
+import { cns } from 'app/utils/cns'
+
+import styles from './CustomBanner.module.css'
+
+const DURATION_BEFORE_SHOW_SURVEY = 2
+const DURATION_UNIT_BEFORE_SHOW_SURVEY: OpUnitType = 'weeks'
+
+function CustomBanner({ message }: { message: any }) {
+  const { value: lastDismissed, set: setLastDismissed } = useLocalStorageValue<
+    string | null
+  >(LocalStorageKeys.SurveyBannerDismissed, { defaultValue: null })
```

Comment:
You should use a different local storage to the survey banner, they would be separate entitities to track when last dismissed